app/_src/guides/federate-kv.md

```diff
@@ -0,0 +1,181 @@
+
+```sh
+helm install --kube-context=mesh-global --create-namespace --namespace {{site.mesh_namespace}} \
+--set {{site.set_flag_values_prefix}}controlPlane.mode=global \
+--set {{site.set_flag_values_prefix}}controlPlane.defaults.skipMeshCreation=true \
+{{ site.mesh_helm_install_name }} {{ site.mesh_helm_repo }}
+```
+
+We skip default mesh creation as we will bring mesh from zone control plane in the next steps.
+
+### Sync endpoint
+
+Find and save the external IP and port of the `{{site.mesh_cp_zone_sync_name_prefix}}global-zone-sync` service in the `{{site.mesh_namespace}}` namespace:
+
+```shell
+export KDS_IP=$(kubectl --context=mesh-global get svc --namespace {{site.mesh_namespace}} {{site.mesh_cp_zone_sync_name_prefix}}global-zone-sync -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+```
+
+{% tip %}
+If you are using minikube, you should use [`host.minikube.internal`](https://minikube.sigs.k8s.io/docs/handbook/host-access/) to ensure networking works correctly.
+
+```sh
+export KDS_IP=host.minikube.internal
+```
+{% endtip %}
+
+## Copy resources from zone to global control plane
+
+To federate zone control plane without any traffic interruption, we need to copy resources like secrets, meshes etc.
+First, we need to expose API server of zone control plane:
+
+```sh
+kubectl --context=mesh-zone port-forward svc/{{site.mesh_cp_name}} -n {{site.mesh_namespace}} 5681:5681
+```
+
+Then we export resources:
+```sh
+export ZONE_USER_ADMIN_TOKEN=$(kubectl --context=mesh-zone get secrets -n {{site.mesh_namespace}} admin-user-token -ojson | jq -r .data.value | base64 -d)
+kumactl config control-planes add \
+  --address http://localhost:5681 \
+  --headers "authorization=Bearer $ZONE_USER_ADMIN_TOKEN" \
+  --name "zone-cp" \
+  --overwrite  
+  
+kumactl export --profile=federation-with-policies --format=kubernetes > resources.yaml
+```
+
+And finally, we apply resources on global control plane
+```sh
+kubectl apply --context=mesh-global -f resources.yaml
+```
+
+## Connect zone control plane to global control plane
+
+Update Helm deployment of zone control plane to configure connection to the global control plane.
+
+
+```sh
+helm upgrade --kube-context=mesh-zone --namespace {{site.mesh_namespace}} \
+--set {{site.set_flag_values_prefix}}controlPlane.mode=zone \
+--set {{site.set_flag_values_prefix}}controlPlane.zone=zone-1 \
+--set {{site.set_flag_values_prefix}}ingress.enabled=true \
+--set {{site.set_flag_values_prefix}}controlPlane.kdsGlobalAddress=grpcs://${KDS_IP}:5685 \
+--set {{site.set_flag_values_prefix}}controlPlane.tls.kdsZoneClient.skipVerify=true \
+{{ site.mesh_helm_install_name }} {{ site.mesh_helm_repo }}
+```
+
+## Verify federation
+
+To verify federation, first port-forward the API service from the global control plane to port 15681 to avoid collision with previous port forward. 
+
+```sh
+kubectl --context=mesh-global port-forward svc/{{site.mesh_cp_name}} -n {{site.mesh_namespace}} 15681:5681
```

Comment:
Let's be consistent and not use `=` after the flag names whenever it's unnecessary
```suggestion
kubectl --context mesh-global port-forward svc/{{site.mesh_cp_name}} -n {{site.mesh_namespace}} 15681:5681
```